ldap/src/integration-test/java/org/springframework/security/ldap/server/UnboundIdContainerTests.java

```diff
@@ -77,4 +84,97 @@ public void afterPropertiesSetWhenPortIsZeroThenRandomPortIsSelected() throws Ex
 		}
 	}
 
+	@Test
```

Comment:
A few of these new tests are failing. You can run `./gradlew clean build` before committing, to ensure that all tests are passing.

Reply:
Sorry my machine crashed and I didn't wanted to lose the work.